src/frontends/common/include/openvino/frontend/extension/op.hpp

```diff
@@ -198,6 +218,85 @@ class OpConversionFunction {
     std::map<std::string, ov::Any> m_attr_values_map;
 };
 
+class FWVisitorInputAttributes : public ov::AttributeVisitor {
+public:
+    explicit FWVisitorInputAttributes(const NodeContext& context,
+                                      const std::map<std::string, size_t>& attr_names_map = {},
+                                      const std::map<std::string, ov::Any>& attr_values_map = {})
+        : m_context(context),
+          m_attr_names_map(attr_names_map),
+          m_attr_values_map(attr_values_map) {}
+
+    void on_adapter(const std::string& name, ValueAccessor<void>& adapter) override {
+        auto p_value = m_attr_values_map.find(name);
+
+        if (p_value != m_attr_values_map.end()) {
```

Comment:
it looks like the logic is incorrect
the value will override the mapping, I'm fixing this in PR
file: https://github.com/openvinotoolkit/openvino/pull/15294/ 
src/frontends/common/include/openvino/frontend/extension/op.hpp

we agreed with Sergey L. that if both the value and name mapping are provided, this is an issue and the behavior is undefined

Reply:
done